features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -1877,6 +1877,14 @@ CREATE TRIGGER IDV_CLAIM_TRIG NO CASCADE
             END
 /
 
+CREATE TABLE IDN_OAUTH_PAR (
+    REQ_URI_UUID VARCHAR(255) PRIMARY KEY,
+    CLIENT_ID VARCHAR(255),
+    SCHEDULED_EXPIRY BIGINT,
+    JSON_PARAMS CLOB(1M)
```

Comment:
Lets remove the JSON part from JSON_PARAMS. We dont need to have the data type here